### PR TITLE
AdOcean network integration  (AdOcean sp. z o.o.)

### PR DESCRIPTION
**Pull request by jedrzej-jelinek-gemius**
AdOcean network integration for AMP project.
CLA was signed by the company AdOcean sp. z  o.o. and Jacek Grabowski was the signer.

### CODE REVIEW

File: ads/_config.js [hunk 1]
```diff
@@ -105,6 +105,8 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  adocean: {},
```
Comment (lannka) [inline ads/_config.js]: could you implement renderStart API? check out the [guidelines](https://github.com/ampproject/amphtml/blob/master/ads/README.md#developer-guidelines-for-a-pull-request)

Comment (jedrzej-jelinek-gemius) [inline ads/_config.js]: Guidelines suggests that renderStart API implementation is optional. We have planned to implement renderStart API together with ad-resize implementation in a near future.

File: ads/adocean.js [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+/**
+ * @param {!Window} global
+ * @param {!function()} cb
+ */
+function runSync(global, cb) {
+  global['__aoPrivFnct' + ++runSyncCount] = cb;
+  /*eslint no-useless-concat: 0*/
+  global.document
+      .write('<' + 'script>__aoPrivFnct' + runSyncCount + '();<' + '/script>');
```
Comment (lannka) [inline ads/adocean.js]: for my own education, can you explain a bit why this is different from
```
 cb();
```

File: ads/_config.js (hunk 1 above)
Comment (lannka) [inline ads/_config.js]: SG

File: ads/adocean.js (hunk 2 above)
Comment (jdrobiecki) [inline ads/adocean.js]: consider code:
```javascript
function cb1 () {
  document.write("<script src='external.js'/>");
}
function cb2 () {
  functionDefinedInExternalJs();
}
```

**Comment by jedrzej-jelinek-gemius** [comment]
You're right, the targeting was set. I've changed  placement ID used in examples, now it should works properly.

**Comment by jedrzej-jelinek-gemius** [comment]
Great, thanks. 

We're working on it. The company must sign CLA, we're waiting for that.

**Comment by jedrzej-jelinek-gemius** [comment]
I signed it!

**Pull request merged**